metadata.md

```diff
@@ -1,45 +1,55 @@
-# Describers: CKAN/Metadata
+# Describers: Ckan/Metadata
```

Comment:
Ah, goodness, someone wanted to remove all the CKAN capitalization, so I said yes, without double-checking. So I guess we need to change them all back now.